problemsets-mat223/modules/module4.tex

```diff
@@ -137,8 +137,20 @@
 \begin{example}
 	Let $\vec a=\mat{1\\2}$, $\vec b=\mat{3\\3}$, $\vec c=\mat{2\\1}$, and $\vec v=\mat{3\\4}$. Which vector out of
 	$\vec a$, $\vec b$, and $\vec c$ has a direction closest to the direction of $\vec v$?
-
-	XXX Finish
+    
+    since $\cos \theta$ is a monotonically decreasing function for $\theta\in[0,\pi]$, we only need to compute the cosines of angles $\alpha$, $\beta$ and $\gamma$ between $\vec a$, $\vec b$, $\vec c$ and $\vec v$ respectively, and find the largest one.
```

Comment:
Haha, slow down there...You're supposed to be explaining this to MAT223 students, not MAT240 students.

Also, capitalize `since` and break this sentence into multiple lines.